dspace-api/src/test/java/org/dspace/authorize/AuthorizeServiceTest.java

```diff
@@ -125,4 +128,39 @@ public void testauthorizeMethodRespectSpecialGroups() {
             throw new AssertionError(ex);
         }
     }
+
+    @Test
+    public void testIsCollectionAdmin() throws SQLException, AuthorizeException {
+
+        context.turnOffAuthorisationSystem();
+
+        Community community = communityService.create(null, context);
+        Collection collection = collectionService.create(context, community);
+        EPerson eperson = ePersonService.create(context);
```

Comment:
Both of these tests should be using `Builder` classes to create all content.  Using a `Builder` will ensure that the content created is *automatically* deleted after the test completes.  Currently, these tests are not cleaning up after themselves.

So, please update these tests to use `CollectionBuilder`, `CommunityBuilder` and `EPersonBuilder`.  Note that you also don't need to create the Admin group, as you can use `CollectionBuilder.withAdminGroup(Eperson)` instead.